src/rez/utils/filesystem.py

```diff
@@ -493,6 +493,19 @@ def safe_chmod(path, mode):
         os.chmod(path, mode)
 
 
+def to_environment_name(name):
```

Comment:
I'm not entirely sure this should be in the filesystem sub-package.

Reply:
Wonder if it would be better to have it in https://github.com/AcademySoftwareFoundation/rez/blob/master/src/rez/utils/formatting.py ?

Reply:
Agreed it is not related to filesystem.

My vote would be to add classmethod `ResolvedContext.pkg_name_to_envvar_name`.

Reply:
`ResolvedContext` is a crowded class. Not sure I would put it as a class method there. But I do understand why you are suggesting that. `REZ_*` environment variables are only used from within a resolved context. I'd still vote for a utils like module like @instinct-vfx suggested (though maybe not formatting?).

Reply:
I don't love it either tbh. Let's go with utils/formatting.py..